layouts/partials/docs/feedback.html

```diff
@@ -0,0 +1,42 @@
+<div id="docsFeedbackContainer" class="mt-8 md:mt-0 hidden">
+    <h4 class="no-anchor text-gray-600 uppercase tracking-wide font-bold text-sm">Feedback</h4>
+    <div id="feedbackButtons">
+        <p class="mt-3 mb-5 text-sm">Was this page helpful?</p>
+        <div class="flex">
+            <a id="docsFeedbackYes" data-track="feedback-yes" class="btn flex">
```

Comment:
The buttons are too visually prominent, IMO (on part with CTAs) -- lemme see if I can come up with a quick treatment that makes them easily discoverable but not quite so demanding of attention.

Reply:
I also wonder if they ought to go just above the "ON THIS PAGE" (when smaller), again so they aren't lost -- e.g., on the programming model page, they're still below the fold.

Might also need to adjust a few margins here and there.